osu.Game.Tests/Visual/Editing/TestSceneEditorSaving.cs

```diff
@@ -24,6 +24,7 @@ public class TestSceneEditorSaving : EditorSavingTestScene
         public void TestCantExitWithoutSaving()
         {
             AddRepeatStep("Exit", () => InputManager.Key(Key.Escape), 10);
+            AddAssert("Sample playback disabled", () => Editor.SamplePlaybackDisabled.Value);
```

Comment:
This still passes on `master`. Please make sure to check tests actually fail without the fix.